### PR TITLE
payoff modelling tutorial: clarify how to use the correct `and` operator

### PR DESCRIPTION
**Pull request by matteolimberto-da**

### CODE REVIEW

File: docs/2.8.0/docs/daml-finance/tutorials/payoff-modeling/basic-builders.rst [hunk 1]
```diff
@@ -120,6 +120,19 @@ When the claim is lifecycled, we obtain
 
 When you want to additively combine more than two claims, you can use the ``andList`` constructor.
 
+.. warning::
+
+  By default, the ``and`` operator is the :ref:`function <function-da-internal-prelude-and-20777>`
+  defined in the Daml standard library. In order to use the claim constructor instead, one needs to 
+  use a qualified import. Alternatively, one can hide the function from the standard library  by
```
Comment (markus-da) [inline docs/2.8.0/docs/daml-finance/tutorials/payoff-modeling/basic-builders.rst:127]:
```suggestion
  use a qualified import. Alternatively, one can hide the function from the standard library by
```

**Review by markus-da — APPROVED** [review]
lgtm, left a minor comment

File: docs/2.8.0/docs/daml-finance/tutorials/payoff-modeling/basic-builders.rst [hunk 2]
```diff
@@ -120,6 +120,19 @@ When the claim is lifecycled, we obtain
 
 When you want to additively combine more than two claims, you can use the ``andList`` constructor.
 
+.. warning::
+
+  By default, the ``and`` operator is the :ref:`function <function-da-internal-prelude-and-20777>`
+  defined in the Daml standard library. In order to use the claim constructor instead, one needs to 
+  use a qualified import. Alternatively, one can hide the function from the standard library by
+  using the syntax
```
Comment (carrielaben-da) [inline docs/2.8.0/docs/daml-finance/tutorials/payoff-modeling/basic-builders.rst:128]:
```suggestion
  defined in the Daml standard library. To use the claim constructor instead, use a qualified import. Alternatively, you can hide the function from the standard library by
  using the syntax
```

**Review by carrielaben-da — CHANGES_REQUESTED** [review]
A couple of changes

**Comment by matteolimberto-da** [comment]
> A couple of changes

Thanks, applied

**Review by carrielaben-da — APPROVED** [review]

**Pull request merged**